platforms/corda-ent/charts/signer/templates/service.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.nodeName }}
+  namespace: {{ .Values.metadata.namespace }}
+  {{ if $.Values.ambassador }}
+  annotations:
+    getambassador.io/config: | 
+      ---
+      apiVersion: ambassador/v1
+      kind: Mapping
+      name: {{ .Values.nodeName }}_mapping
+      prefix: /
+      service: {{ .Values.nodeName }}.{{ .Values.metadata.namespace }}:{{ .Values.service.port }}
+      host: {{ .Values.nodeName }}.{{ .Values.ambassador.external_url_suffix }}:8443
```

Comment:
This may not work until the certificates are generated.

Reply:
Commented for now.